main.ts

```diff
@@ -481,7 +490,7 @@ ipc.on('maximize-window', function (event, someMessage) {
 /**
  * Method to replace thumbnail of a particular item
  */
-ipc.on('replace-thumbnail', function(event, pathToIncomingJpg: string, item: ImageElement) {
+ipc.on('replace-thumbnail', function (event, pathToIncomingJpg: string, item: ImageElement) {
```

Comment:
There's some small formatting here, but I think it would be best to configure `.editorconfig` as discussed and go through the whole codebase in a seperate PR. (Not signing you up for this @ErikDvorcak 😅 )

Don't worry about fixing this for now. 😄